java/example_code/elastictranscoder/src/main/java/aws/example/elastictranscoder/RetryRequest.java

```diff
@@ -0,0 +1,41 @@
+//snippet-sourcedescription:[RetryRequest.java demonstrates how to retry a request.]
+//snippet-keyword:[Java]
+//snippet-keyword:[Code Sample]
+//snippet-keyword:[Amazon Elastic Transcoder]
+//snippet-service:[elastictranscoder]
+//snippet-sourcetype:[snippet]
+//snippet-sourcedate:[]
+//snippet-sourceauthor:[]
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+currentRetry = 0
+DO
```

Comment:
i don't think this is a snippet for java. is it supposed to be part of a different language PR?